### DISCUSSION TITLE
What settings can I do for best results? My voice sounds choppy + robotic on NVIDIA T4 - Please Help

### DISCUSSION THREAD

**Discussion by jzimerman** [w-okada/voice-changer#1374] [General]
I'm running this on google collab with a T4, and I can't seem to get this to sound decent. 

My settings are:
inputChunkNum: 110,
extraConvertSize: 131072,
noiseSuppression: false,
noiseSuppression2: true,

The model I have is well trained and should be performing good.

Do you have any recommendations for settings? What settings are you using? **Am I missing something**?

**Comment by AnnoyingFox** [comment]
It's very hard to tell why would sound good or bad a model. Did you try your model on a computer instead of Google colab to see if there is a difference? Why exactly do you think it sounds bad? Maybe you could upload a little sound file of how it sounds so we can check and see what could be the issue.

**Reply by jzimerman** [reply]
The model is one I took from a YT video of someone running it locally without any issues. 
It sounds like there's audio clipping / robotic despite having the GPU resources for it. Is there a baseline settings that you recommend assuming that the GPU is powerful to handle any settings version?

**Comment by AnnoyingFox** [comment]
Could you share a screenshot of your settings?

**Reply by jzimerman** [reply]
Sure, here are the settings (I took away the frontend)
   numTrancateTreshold: 100,
        volTrancateThreshold: 0.0005,
        volTrancateLength: 32,
        protocol: "sio",
        sendingSampleRate: 48000,
        inputChunkNum: 105,
        downSamplingMode: "average",
        sampleRate: 48000,
        echoCancel: true,
        noiseSuppression: false,
        noiseSuppression2: true,
        inputGain: 1.0,
        outputGain: 3.0,
        monitorGain: 1.0,
        passThroughConfirmationSkip: false
f0Factor: 1.0,
f0Detector: F0Detector.rmvpe_onnx,

**Comment by AnnoyingFox** [comment]
There are some settings that may cause your issue.
For male to female voice conversion, F0 should be around 12. For female to male should be around -12. If it's male to male or female to female, it depends on your pitch but it could be between -4 and 4. So, unless the voice does not need F0 Tune setting, it may not work properly at a value of 1 as it may sound robotic. About the clipping that you mention, having an output gain of 3.0 may cause a too high volume output and may be causing the clipping maybe. It's hard to know without any type of example of how it sounds vs how it should sound. Does the included voices works fine for you?

**Comment by AnnoyingFox** [comment]
It could also be that, since you took off the frontend, something may have broken. Without an UI showing the configuration properly, it's hard to tell.

**Reply by jzimerman** [reply]
Ty I will provide clips when I get home. I appreciate the help so far. I'm just trying to figure out if im doing something wrong with the settings
